examples/quickstart_examples/quickstart_example.py

```
@@ -0,0 +1,215 @@
+    """
+    print("\n--- Model Training Example ---\n")
+
+    # Create synthetic dataset
+    num_graphs = 50
+    num_nodes = 10
+    num_node_features = 16
+    num_edge_features = 4
+
+    # Create random graphs
+    graphs = []
+    for i in range(num_graphs):
```

Ruff (0.11.9)

Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)